languages/de.json

```diff
@@ -1360,6 +1434,7 @@
 "DND5E.SkillBonuses": "Fertigkeitsboni",
 "DND5E.SkillConfigurationHint": "Übung in Fertigkeit und Boni bearbeiten.",
 "DND5E.SkillConfigure": "Fertigkeit bearbeiten",
+"DND5E.SkillsConfig": "Fertigkeit bearbeiten",
```

Comment:
Considering this, in contrast to the preceding one, is plural, it maybe should be "Fertigkeiten"? Not sure, would need to check in context.